pithos/plugins/notify.py

```diff
@@ -45,6 +45,18 @@ def on_prepare(self):
         if 'action-icons' in caps:
             self.notification.set_hint('action-icons', GLib.Variant('b', True))
 
+        # GNOME Shell 3.20 or higher has bultin MPRIS functionality that makes
+        # persistent song notifications redundant.
+        has_built_in_mpris = False
+        version = server_info['version'].split('.')
```

Comment:
If the server follows the spec. Otherwise I would expect they would return empty strings if all else fails.